examples/born/apbs-mol-auto.in

```diff
@@ -4,7 +4,7 @@
 
 # READ IN MOLECULES
 read                                                
-    mol xml ion.xml
+    mol pqr ion.pqr
```

Comment:
Did we decide to drop XML support?  I think we did but I wanted to make sure.

Reply:
I can't find anything official. The reason I changed it was that the same data was in the ion.pqr file that was in the ion.xml file.
I thought we made the decision to support/read PQR now and move to mmCIF/PDBx file formats in the future.

Reply:
Yes, I'm 100% in support of dropping XML-format input in lieu of CIF.  I just wanted to make sure that it was a deliberate decision.